gitprofile.config.js

```diff
@@ -83,6 +83,10 @@ const config = {
     },
   ],
 
+  // Optional custom footer. See README file, section "custom-footer", for details
+  /*
+  footer: '[...]',
+  */
```

Comment:
There should not be any default footer. The user should be able to not display the footer if he chooses to.

Reply:
Just to be 100% sure: do you mean that if no footer is configured in the config file, then no footer will be rendered in the page? Correct?
And, if a user wants to show *"Made with GitProfile and ❤️"*, then they have to configure it explicitly?

Reply:
Yes, that's right.